python/lib/bidsreader.py

```diff
@@ -10,15 +10,24 @@
 
 import lib.exitcode
 import lib.utilities as utilities
+import bids
+
 try:
-    from bids.layout import BIDSLayout
+    from bids import BIDSLayout
```

Comment:
```suggestion
try:
    from bids import BIDSLayout
except ImportError:
    print("Could not import BIDSLayout. Ensure you have PyBIDS >= 0.12.1 installed.")        
    exit(lib.exitcode.INVALID_IMPORT)
```

If we plan on supporting only PyBIDS > 0.12.1, we should add an error message to tell users to update their pybids package.

Reply:
The change is not supposed to break Pybids < 0.12.1 since I check the version before applying the changes.